### DISCUSSION TITLE
Ability to delete WordPress core Role

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Capabilities#682] [Feature Requests]
> I also discovered that I don’t have the native possibility to hide or delete the roles not necessary for my case “subscriber and contributor”
> 
> [so I had to try to understand what to do](https://wordpress.stackexchange.com/questions/235331/how-to-delete-user-roles) and I installed this [plugin](https://wordpress.org/plugins/members/) to be able to delete the roles then I uninstalled it.
> 
> it was very unpleasant to have to do it with another plugin
> I imagine that one day this possibility will be included in native with publishpress capabilities 🙏

Original thread: https://wordpress.org/support/topic/ability-to-disable-multiple-roles-by-default/

**Comment by olatechpro** [comment]
@rizaardiyanto1412 Is this issue same as https://github.com/publishpress/PublishPress-Capabilities/issues/622?

I noticed you linked to same WordPress issue but this issue title is misleading as ability to delete wordpress core role is definitely not a feature we'll want to add

**Comment by rizaardiyanto1412** [comment]
@olatechpro The ticket is the same but he requested 2 featured, #622 and this one. 

Unfortunately yes, that's what he wants. He wants to be able to delete WP core role (subscribe, contributor) which he did not use. 

So I guess the answer we will not make it happen right?

**Comment by olatechpro** [comment]
### Slack comments
**Steve**:  Good question. Thanks for pushing back on that
I don't fully know the implications for deleting some lower-level roles
Certainly it's not an urgent issue and we can easily postpone to the next milestone

**Ola**: Well, if you don’t see problem with allowing deletion of system roles, then I don’t have personal issue with it as well.
I raised the question as everything we’ve done so far in relation to role doesn’t really tamper with system role.
If we’re going ahead with this, what exactly is the implementation usage?
Are we simply adding ‘contributor’, and ‘subscriber’ to roles that can be deleted as when i was reading it, i was reading it as allowing system roles deletion but you made mention of “deleting some lower-level roles” which shows you have specific roles in mind that should be allowed for deletion

**Steve**: To me, I think we should mark this as "discussion needed" and move to the next milestone
We can add these notes and thoughts to the issue, and maybe Kevin Behrens can contribute later

**Ola**: Okay, that make sense
To me, in general, allowing system roles deletion not good but maybe not really bad if we limit the roles to selected lower level roles like you mentioned and test to be sure we’re not causing any problem.
